util/app-config/src/cli.rs

```diff
@@ -526,3 +526,12 @@ fn is_hex(hex: &str) -> Result<(), String> {
         Err("Must 0x-prefixed hexadecimal string".to_string())
     }
 }
+
+fn is_h256(hex: &str) -> Result<(), String> {
+    let tmp = hex.as_bytes();
+    if tmp[..2] == b"0x"[..] && tmp.len() == 66 {
+        is_hex(hex)
+    } else {
+        Err("Must be 0x-prefixed hexadecimal string and string length is 66".to_owned())
+    }
```

Comment:
`is_hex` fn has validated the input string already, only need to add an extra len checking here:

```suggestion
    if hex.len() != 66 {
        Err("Must be 0x-prefixed hexadecimal string and string length is 66".to_owned())
    } else {
        is_hex(hex)
    }
```

Reply:
done!